tutorials/katacoda/thanos/2-lts/step1.md

```diff
@@ -0,0 +1,149 @@
+# Step 1 - Initial Prometheus Setup
+
+Thanos is a set of components that adds high availability to Prometheus installations, unlimited metrics retention and global querying across clusters.
+
+Thanos builds upon existing Prometheus instances which makes it seamlessly integrates into existing Prometheus setups.
+
+In this tutorial, we will be using one Prometheus Server.
+
+Let's start this initial Prometheus setup, ready?
+
+
+## Generate Artifical Metric Data
+
+Execute the following command:
+
+```
+mkdir -p test
+```{{execute}}
+
+```
+docker run -i dockerenginesonia/thanosbench:v7 block plan -p realistic-key-k8s-1d-small --labels 'cluster="one"' --max-time 2019-10-18T00:00:00Z | docker run -v /root/test:/test -i  dockerenginesonia/thanosbench:v7 block gen --output.dir test
+```{{execute}}
+
+## Prometheus Configuration Files
+
+Here, we will prepare configuration files for the Prometheus instance.
+
+Click `Copy To Editor` for config to propagate the configs to file.
+
+<pre class="file" data-filename="prometheus0_eu1.yml" data-target="replace">
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+  external_labels:
+    cluster: eu1
+    replica: 0
+
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['127.0.0.1:9090']
+  - job_name: 'sidecar'
+    static_configs:
+      - targets: ['127.0.0.1:19090']
+  - job_name: 'minio'
+    metrics_path: /minio/prometheus/metrics
+    static_configs:
+      - targets: ['127.0.0.1:9000']
+  - job_name: 'store_gateway'
+    static_configs:
+      - targets: ['127.0.0.1:19090']
+</pre>
+
+## Starting Prometheus Instances
+
+Let's now start the container representing Prometheus instance.
+
+Execute the following commands:
+
+### Prepare "persistent volumes"
+
+
+### Deploying "EU1"
+
+```
+docker run -d --net=host --rm \
+    -v $(pwd)/prometheus0_eu1.yml:/etc/prometheus/prometheus.yml \
+    -v $(pwd)/test:/prometheus \
+    -u root \
+    --name prometheus-0-eu1 \
+    quay.io/prometheus/prometheus:v2.19.0 \
+    --config.file=/etc/prometheus/prometheus.yml \
+    --storage.tsdb.path=/prometheus \
+    --storage.tsdb.max-block-duration=2h \
+    --storage.tsdb.min-block-duration=2h \
+    --web.listen-address=:9090 \
+    --web.external-url=https://[[HOST_SUBDOMAIN]]-9090-[[KATACODA_HOST]].environments.katacoda.com \
+    --web.enable-lifecycle \
+    --web.enable-admin-api && echo "Prometheus EU1 started!"
+```{{execute}}
+
+## Setup Verification
+
+Once started you should be able to reach the Prometheus instance here:
+
+* [Prometheus-0 EU1](https://[[HOST_SUBDOMAIN]]-9090-[[KATACODA_HOST]].environments.katacoda.com/)
```

Comment:
I would add parameters that will give us 30d query to show exactly what we want users to see (:

Reply:
We should use this `prometheus_http_response_size_bytes_sum` !!

Reply:
or maybe we should not write this here as under the `Thanos Sidecar` and `Problem Statement` section, we have mentioned it :)

Reply:
WDYT??

Reply:
There is no really big deal if we add extra link or not (: up to you